test/test_helper.js

```diff
@@ -0,0 +1,4 @@
+process.env.NODE_ENV = 'test'
+require('../app.js')
```

Comment:
`express.listen` seems to fail silently if the port is already attached, which means it's possible the test suite will be run against some other app attached to 3000. A global `before` in test_helper.js could quickly check for this situation like this:

```js
// Make sure nothing else bound to port:
before(function () {
  return require('request').get('http://localhost:' + config.get('port'), function (err, response, body) {
    if (err && err.code === 'ECONNREFUSED') {
      // No test server is running, so safe to instantiate one from local code:
      require('../app.js')
    } else {
      console.err(`Something else already listening on ${config.get('port')}. Aborting.`)
      process.exit()
    }   
  })  
})```